### DISCUSSION TITLE
Not work with external database for image alpine

### DISCUSSION THREAD

**Discussion by syji35** [dani-garcia/vaultwarden#3518] [Q&A]
With version 1.28.1-alpine of the docker image, it is impossible to connect to an external database. It works with the non-alpine image

Here my logs : 

'''
2023-05-10 13:56:06.955][vaultwarden::util][WARN] Can't connect to database, retrying: DieselCon.

[CAUSE] BadConnection(

    "Unknown server host '<my_host>' (-2)",

)
'''

**Comment by BlackDex** [comment]
It could be that the alpine image isn't able to get the correct IP based upon the hostname.
But that shouldn't be the issue. Could  you provide a bit more details on how you deployed the instance?
docker-compose file for example, the Vaultwarden config on how you try to connect to the database etc.. etc..
